pkg/service/utils.go

```diff
@@ -0,0 +1,110 @@
+package service
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"github.com/coreos/go-systemd/v22/unit"
+)
+
+// CompareServices compare 2 service and return true if serviceA has all the fields of serviceB
+func CompareServices(serviceA, serviceB *Service) (bool, error) {
+	optsA, err := unit.Deserialize(strings.NewReader(serviceA.Content))
+	if err != nil {
+		return false, err
+	}
+	optsB, err := unit.Deserialize(strings.NewReader(serviceB.Content))
+	if err != nil {
+		return false, err
+	}
+
+	for _, optB := range optsB {
+		found := false
+		for _, optA := range optsA {
+			if optA.Match(optB) {
+				found = true
+				break
+			}
+		}
+
+		if !found {
```

Comment:
so after reading about the break label I think what you did first with the outer was better to break from both loops.